### PR TITLE
Additional QrProperties

### PR DESCRIPTION
**Pull request by CaseyODonnell92**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.electrum</groupId>
   <artifactId>qr-payment-service-interface</artifactId>
-  <version>${major-version}.${minor-version}.${patch-version}</version>
+  <version>${major-version}.${minor-version}.${patch-version}-RC1</version>
```
Comment (TimElectrum) [inline pom.xml:7]: reminder about this

File: src/main/java/io/electrum/qr/api/model/QrProperties.java [hunk 2]
```diff
@@ -148,6 +162,53 @@ public void setExpiryDate(OffsetDateTime expiryDate) {
...
    }
 
+   public QrProperties customerProvidedValuePrompts(List<CustomerProvidedValuePrompt> customerProvidedValuePrompts) {
+      this.customerProvidedValuePrompts = customerProvidedValuePrompts;
+      return this;
+   }
+
+   /**
+    * Details for prompts that can be displayed to the customer by a scanning application which prompt the customer to
+    * provide user-provided text values which can be used when performing actions using the QR code's associated data.
```
Comment (TimElectrum) [inline src/main/java/io/electrum/qr/api/model/QrProperties.java:172]: this reads a little word-y. Perhaps it just needs an extra full stop in there or something but would you mind revising this, please?

**Review by TimElectrum — COMMENTED** [review]
happy enough with it but maybe revisit the description of the prompts.

File: src/main/java/io/electrum/qr/api/model/QrProperties.java (hunk 2 above)
Comment (mattsegers) [reply src/main/java/io/electrum/qr/api/model/QrProperties.java]: much easier to read now :)

**Review by mattsegers — APPROVED** [review]

**Pull request merged**